eva_submission/submission_qc_checks.py

```diff
@@ -304,6 +470,10 @@ def run_qc_checks_for_submission(self):
         Accessioning job check: {self._accessioning_job_check_result}
         Variants Skipped accessioning check: {self._variants_skipped_accessioning_check_result}
         Variant load check: {self._variant_load_job_check_result}
+        Remapping and Clustering Check: 
+            Clustering check: {self._clustering_check_result} 
+            Remapping Ingestion check: {self._remapping_check_result}
```

Comment:
```suggestion
            Remapping check: {self._remapping_check_result}
```
As (I think) it includes both extraction and ingestion errors.

Reply:
updated